### ISSUE TITLE
Please provide cvdump.pdb as well.

### ISSUE THREAD

**Issue by EricYoungdale** [Microsoft/microsoft-pdb#20]
You provide the executable, which is nice, but as long as things are in a state where we cannot build the thing ourselves, providing the PDB for the executable that you do include would at least let us step through and get a better feel for how it works.

Providing both debug and release builds of the tool might be useful as well.

Right now I am looking at an issue where cvdump says the TYPES stream is corrupt.  Why?  Hard to say - I am having to step through in assembly language to try and deduce why this might be the case, and as you might expect this is a tedious process.

**Comment by jonwil** [comment]
Yes please, being able to step through cvdump source code in the debugger would be very useful :)
